pages/[mentorUsername].page.js

```diff
@@ -107,58 +113,34 @@ function Index({ mentorDetail }) {
   return (
     <>
       <React.Suspense fallback={<div>Loading...</div>}>
+
         <Head>
           <title>GrabTern | Book Your Session</title>
         </Head>
         <Header navbarBackground={true} />
         {/* Mentor Page */}
         <main className="tw-flex tw-flex-col tw-items-center">
-          <div className="tw-flex tw-flex-row tw-justify-center tw-items-start tw-my-44 tw-gap-[60px] tw-flex-wrap">
-            <MentorCard
-              mentorImage={mentorDetail.mentorImg}
-              name={mentorDetail.name}
-              internAt={mentorDetail.internAt}
-              currentStatus={mentorDetail.currentStatus}
-              socialLinks={mentorDetail.social}
-              about={mentorDetail.description}
-              handleSharePage={() => setShowModal(true)}
-            />
-            {/* Session Cards Container */}
-            <div className="tw-flex tw-flex-col tw-items-stretch tw-max-w-[448px]">
-              {/* Session Cards for every session */}
-              {mentorDetail?.bookSession?.length !== 0 &&
-                mentorDetail?.bookSession?.map((session, index) => (
-                  <SessionCard
-                    key={index}
-                    type={session?.sessionType}
-                    name={session?.sessionName}
-                    description={session?.sessionDescription}
-                    duration={session?.sessionMeetingDuration}
-                    pricePerSession={session?.priceSession}
-                    handleBookSession={() => {
```

Comment:
as you created new component so, like wise before it is good to import component here and not to remove the funtionality of sessionBooking
@sayyedarib  or we can have a particular booking Session component for it???